Makefile

```
@@ -477,11 +477,17 @@ version: $(VERSRC)
 
 # This rule triggers re-generation of version files specified if Makefile changes.
 $(VERSRC): Makefile
+	@if [ \
+			$$(grep "Version = " version.go | awk -F'Version = "' '{print$$2}' | cut -d "." -f1) \
```

```suggestion
			$$(sed -n 's/.*Version = "\(.*\)".*/\1/p' version.go | cut -d"." -f1) \
```
or just
```suggestion
			$$(awk -F'Version = "' '{printf $$2}' version.go | cut -d"." -f1) \
```
may be slightly less verbose.